Easydict/objc/MMKit/Log/MMLog.h

```diff
@@ -50,12 +50,15 @@ extern DDLogLevel MMDefaultLogLevel;
 extern BOOL MMDefaultLogAsyncEnabled;
 
 
-#define MMLogInfo(frmt, ...) LOG_MAYBE_TO_DDLOG([MMManagerForLog sharedDDLog], MMDefaultLogAsyncEnabled, MMDefaultLogLevel, DDLogFlagInfo, 0, nil, __PRETTY_FUNCTION__, frmt, ##__VA_ARGS__)
 #define MMLogVerbose(frmt, ...) LOG_MAYBE_TO_DDLOG([MMManagerForLog sharedDDLog], MMDefaultLogAsyncEnabled, MMDefaultLogLevel, DDLogFlagVerbose, 0, nil, __PRETTY_FUNCTION__, frmt, ##__VA_ARGS__)
```

Comment:
As we already have the limitation on log file size and disk count, now we can use logs liberally, so can we remove `MMLogVerbose`? I think the three log types are enough: `MMLogInfo`, `MMLogWarn` and `MMLogError`.

Reply:
I prefer to keep `verbose` in case we want to only output some info to the terminal.

Reply:
To change all of the NSLog to `info` may have a problem that some useless info logs are written into the file.  Keeping the `verbose` may be helpful if we only output some info to the terminal.

Reply:
Ok, I have kept the `verbose`, just changed it to `MMLog`.

You are right, I just removed this macro.

```
#define NSLog(frmt, ...) MMLog(frmt, ##__VA_ARGS__)
```

Reply:
Ok, 'NSLog' may not a good idea, then we have two kinds of log interface styles. 

'MMLog' has the same problem IMHO, compare with 'MMLog'/'MMLogInfo'/'MMLogError', I think they are naming in different styles, maybe confused when using them.

Reply:
That is my intention, I want to differentiate `MMLog` from `MMLogInfo` and `MMLogError`, `MMLog` is only used in debug mode.

The `verbose` naming is not as clear as `info`, `error,`  it reminds people of the `--verbose` parameter in the command line. Anyway, I can't tell its meaning at a glance without checking the documentation.

Reply:
One of the simplest ways to understand it is to use `MMLog` just like `NSLog`.

Reply:
OK